src/v2/providers/alerts/index.ts

```diff
@@ -1,5 +1 @@
-import * as crashlytics from './crashlytics';
-
-export { crashlytics };
```

Comment:
why did we yank this?

Reply:
Added the import/export back in the latest commit